spin/cmds/meson.py

```diff
@@ -257,7 +278,19 @@ def build(
     or set CFLAGS appropriately:
 
       CFLAGS="-O0 -g" spin build
+
+    Build into a different build/build-install directory:
+
+      spin build -C build-for-feature-x
+
+    This feature is useful in combination with a shell alias, e.g.:
+
+      $ alias spin-clang="spin -C build-clang"
+
+    Which can then be used to build (`spin-clang build`), to test (`spin-clang test ...`), etc.
```

Comment:
The name `build-clang` might be a bit misleading. To me it implies that this feature can somehow be used to choose a different compiler (clang). As I understand it, you'd need to update the underlying "build" command / configuration which would then apply regardless of build directory? Or am I missing something?